libc/test/UnitTest/FPExceptMatcher.h

```diff
@@ -53,11 +116,42 @@ class FPExceptMatcher : public Matcher<bool> {
 } // namespace testing
 } // namespace LIBC_NAMESPACE
 
+// Matches on the FP exception flag `expected` being *equal* to FP exception
+// flag `actual`
+#define EXPECT_FP_EXCEPT_EQUAL(expected, actual)                               \
```

Comment:
nit: I'd prefer to use `EXCEPTION_EQ` rather than `EXCEPT_EQUAL`. I find having the words `EXPECT` and `EXCEPT` in the same macro name hard to read.

Reply:
Made the changes in 56e8d886106025434eb0289963469600859ebc1d. I've also changed all the other macros with both `EXPECT` and `EXCEPT` in the identifier. After doing this, there was a name collision with `EXPECT_FP_EXCEPTION(exception)` we were previously using. I changed this to `EXPECT_FP_EXCEPTION_HAPPENED(exception)` and changed all uses in the tests, so it's a large commit.